app/models/company_branch.rb

```diff
@@ -0,0 +1,13 @@
+# == Schema Information
+#
+# Table name: company_branches
+#
+#  id         :bigint           not null, primary key
+#  address    :string
+#  name       :string
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
```

Comment:
Why do you have a created_at and a updated_at attribute? just curious since it isn't used in any reports

Reply:
In general, a table should always have these fields in order to have control over the information that is recorded and when it is updated.

Reply:
I see, thanks for the info